### ISSUE TITLE
Using mixed backends for dynamic/static configuration

### ISSUE THREAD

**Issue by mvdstam** [containous/traefik#1649]
Hey all, I'm trying to get the following working.

I want to run Traefik in a clustered environment with Docker (Docker *Swarm*), where Traefik runs on each and every master node in the cluster. Traefik listens for Docker events and updates its internal __dynamic__ configuration accordingly.
In other words, the backend for the *Dynamic* configuration is Docker.

However, Traefik also has a __static__ configuration. I want Traefik to store *that* configuration in a distributed KV store like Zookeeper so I don't have do maintain a copy of a TOML configuration file on each server that Traefik runs on. If I need to change the configuration later on, I can simply change it in Zookeeper and restart the Traefik instances one by one without ever having to change any files at all.
In other words, the backend for the *Static* configuration is Zookeeper.

I've tried to set this up myself, but when the static configuration is stored in Zookeeper (with the configuration having the correct `Docker` configuration settings), Traefik doesn't seem to listen for Docker events anymore. Is this a bug or simply not supported at the moment?

**Comment by mvdstam** [comment]
Ping @emilevauge @vdemeester 😃

**Comment by sebastien-prudhomme** [comment]
Have the same problem mixing Consul, Docker and File backends.

Consul + Docker backends work fine together
Consul + File backends work fine together
Docker + File backends work fine together

If i use the three backends together, the Docker backend is not working

**Comment by mvdstam** [comment]
> Is this a bug or simply not supported at the moment?

@Idez Can provide some clarity in this matter? Happy to provide additional information if necessary, but at this moment we're not really sure if this is a bug or feature request 😅

**Comment by traefiker** [comment]
I'll close the issue due to inactivity but feel free to re-open if needed.
